symbolic-base/src/ZkFold/Base/Algebra/Polynomials/Multivariate/Polynomial.hs

```diff
@@ -8,10 +8,7 @@ import           Control.DeepSeq                                       (NFData)
 import           Data.Aeson                                            (FromJSON, ToJSON)
 import           Data.Bifunctor                                        (Bifunctor (..))
 import           Data.Functor                                          ((<&>))
-import           Data.List                                             (intercalate)
-#if __GLASGOW_HASKELL__ < 912
-import           Data.List                                             (foldl')
-#endif
+import           Data.List                                             (foldl', intercalate)
```

Comment:
I think that, actually, what stylish-haskell did here is ok because for both GHC 9.6 and 9.12, foldl' can be found here

Reply:
But then GHC 9.12 will complain about redundant imports, and we're building with -Werror

Reply:
Oof, ok
Technically, we could rewrite it to not depend on the Prelude (or, at least, list the dependencies explicitly)